docs/Guides/getting-startted/converting-to-mcap.md

```diff
@@ -0,0 +1,25 @@
+# Converting To MCAP
+
+If you already have existing data that is not in the MCAP file format, you may want to **convert this non-MCAP data into MCAP files**.
+
+[Install the mcap CLI tool](https://github.com/foxglove/mcap/tree/main/go/cli/mcap#installing) to start converting your non-MCAP data to MCAP files.
+
+### ROS 1
+
+```
+$ mcap convert ../../testdata/bags/demo.bag demo.mcap
+```
+
+### ROS 2
+
+```
+$ mcap convert multiple_files_1.db3 demo.mcap
+```
+
+mcap will search the path stored in your $AMENT_PREFIX_PATH environment variable to locate the ROS message definitions on your hard drive.
```

Comment:
"The tool will search ..."?

Reply:
or This will search, maybe - mcap the file format isn't searching, so the sentence is a little unclea.r